### ISSUE TITLE
Can't install version > 2.9 on FreeBSD because of `ruby_racer` gem dependency incompatibility

### ISSUE THREAD

**Issue by MacLemon** [zammad/zammad#2616]
### Infos:

* Used Zammad version: 2.9.0 -> 3.0.0
* Installation method (source, package, ..): source, no packages provided by Zammad project
* Operating system: FreeBSD 11.2-p10
* Database + version: PostGreSQL 10.8
* Elasticsearch version: elasticsearch 5.6.8
* Browser + version: Chromium 75, Firefox 67

### Expected behavior:

* Updating via source from 2.9.0 to 3.0.0 should be possible.

### Actual behavior:

* Upgrade doesn't work due to new dependencies

### Steps to reproduce the behavior:

* Following the [Update Documentation](https://docs.zammad.org/en/latest/install-update.html#source-update)

The step
```
zammad@shell> bundle install --without test development mysql
```
fails with the following error message:

```
----------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/ruby/gems/2.5/gems/libv8-6.7.288.46.1/vendor/depot_tools/gclient.py", line 2935, in <module>
    sys.exit(main(sys.argv[1:]))
  File "/usr/local/lib/ruby/gems/2.5/gems/libv8-6.7.288.46.1/vendor/depot_tools/gclient.py", line 2921, in main
    return dispatcher.execute(OptionParser(), argv)
  File "/usr/local/lib/ruby/gems/2.5/gems/libv8-6.7.288.46.1/vendor/depot_tools/subcommand.py", line 252, in execute
    return command(parser, args[1:])
  File "/usr/local/lib/ruby/gems/2.5/gems/libv8-6.7.288.46.1/vendor/depot_tools/gclient.py", line 2563, in CMDsync
    ret = client.RunOnDeps('update', args)
  File "/usr/local/lib/ruby/gems/2.5/gems/libv8-6.7.288.46.1/vendor/depot_tools/gclient.py", line 1546, in RunOnDeps
    patch_refs=patch_refs)
  File "/usr/local/lib/ruby/gems/2.5/gems/libv8-6.7.288.46.1/vendor/depot_tools/gclient_utils.py", line 1075, in run
    self.item.run(*self.args, **self.kwargs)
  File "/usr/local/lib/ruby/gems/2.5/gems/libv8-6.7.288.46.1/vendor/depot_tools/gclient.py", line 892, in run
    self.ParseDepsFile()
  File "/usr/local/lib/ruby/gems/2.5/gems/libv8-6.7.288.46.1/vendor/depot_tools/gclient.py", line 670, in ParseDepsFile
    filepath, self.get_vars())
  File "/usr/local/lib/ruby/gems/2.5/gems/libv8-6.7.288.46.1/vendor/depot_tools/gclient.py", line 1175, in get_vars
    'host_os': _detect_host_os(),
  File "/usr/local/lib/ruby/gems/2.5/gems/libv8-6.7.288.46.1/vendor/depot_tools/gclient.py", line 1209, in _detect_host_os
    return _PLATFORM_MAPPING[sys.platform]
KeyError: 'freebsd11'
Running: gclient root
Running: gclient config --spec 'solutions = [
  {
    "url": "https://chromium.googlesource.com/v8/v8.git",
    "managed": False,
    "name": "v8",
    "deps_file": "DEPS",
    "custom_deps": {},
  },
]
'
Running: gclient sync --with_branch_heads
Traceback (most recent call last):
  File "/usr/local/lib/ruby/gems/2.5/gems/libv8-6.7.288.46.1/vendor/depot_tools/fetch.py", line 306, in <module>
    sys.exit(main())
  File "/usr/local/lib/ruby/gems/2.5/gems/libv8-6.7.288.46.1/vendor/depot_tools/fetch.py", line 301, in main
    return run(options, spec, root)
  File "/usr/local/lib/ruby/gems/2.5/gems/libv8-6.7.288.46.1/vendor/depot_tools/fetch.py", line 295, in run
    return checkout.init()
  File "/usr/local/lib/ruby/gems/2.5/gems/libv8-6.7.288.46.1/vendor/depot_tools/fetch.py", line 137, in init
    self.run_gclient(*sync_cmd)
  File "/usr/local/lib/ruby/gems/2.5/gems/libv8-6.7.288.46.1/vendor/depot_tools/fetch.py", line 82, in run_gclient
    return self.run(cmd_prefix + cmd, **kwargs)
  File "/usr/local/lib/ruby/gems/2.5/gems/libv8-6.7.288.46.1/vendor/depot_tools/fetch.py", line 71, in run
    subprocess.check_call(cmd, **kwargs)
  File "/usr/local/lib/python2.7/subprocess.py", line 190, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '('gclient', 'sync', '--with_branch_heads')' returned non-zero exit status 1
/usr/local/lib/ruby/gems/2.5/gems/libv8-6.7.288.46.1/ext/libv8/builder.rb:79:in `block in setup_build_deps!': unable to fetch v8 source (RuntimeError)
        from /usr/local/lib/ruby/gems/2.5/gems/libv8-6.7.288.46.1/ext/libv8/builder.rb:77:in `chdir'
        from /usr/local/lib/ruby/gems/2.5/gems/libv8-6.7.288.46.1/ext/libv8/builder.rb:77:in `setup_build_deps!'
        from /usr/local/lib/ruby/gems/2.5/gems/libv8-6.7.288.46.1/ext/libv8/builder.rb:36:in `build_libv8!'
        from /usr/local/lib/ruby/gems/2.5/gems/libv8-6.7.288.46.1/ext/libv8/location.rb:24:in `install!'
        from extconf.rb:7:in `<main>'

extconf failed, exit code 1

Gem files will remain installed in /usr/local/lib/ruby/gems/2.5/gems/libv8-6.7.288.46.1 for inspection.
Results logged to /usr/local/lib/ruby/gems/2.5/extensions/amd64-freebsd-11/2.5/libv8-6.7.288.46.1/gem_make.out

An error occurred while installing libv8 (6.7.288.46.1), and Bundler cannot continue.
Make sure that `gem install libv8 -v '6.7.288.46.1' --source 'https://rubygems.org/'` succeeds before bundling.

In Gemfile:
  mini_racer was resolved to 0.2.4, which depends on
    libv8
```

### Regression
Up until Zammad 2.9.0 installation from source on FreeBSD has been working fine and we've been running it successfully for over two years now.

Yes I'm sure this is a bug and no feature request or a general question.

**Comment by SummerStorm** [comment]
Hi, there. Thanks for the report. Please try these steps:

Step 1. Apply this patch:
```diff
diff --git a/Gemfile b/Gemfile
index 714fbff06..c5cc6c8c4 100644
--- a/Gemfile
+++ b/Gemfile
@@ -58,2 +58,2 @@ gem 'execjs'
-gem 'libv8'
-gem 'mini_racer'
+gem 'libv8', '~> 3.11.8'
+gem 'therubyracer'
```

Step 2. Run `bundle install --without test development mysql` again.

Unfortunately I don't have a FreeBSD system to replicate this, but I'm fairly certain this will work. Once it does, we'll look into a more permanent fix. If it doesn't, please post the error output and I'll help you with the next step.

###  Background info for developers

Commit 3acfa4a277c29a0eb3b7bff83fde1a60718a9746 updated from therubyracer (0.11.4) and libv8 (3.11.8.17) to the new gems of mini_racer (0.2.4) and libv8 (6.7.288.46.1). Note the huge v8 JS engine version bump. I believe this was mainly done for performance reasons, and it wasn't necessarily required to fix issue #2089. Basically therubyracer is an older gem that depended on an older version of the V8 engine, while mini_racer is a newer fully compatible replacement that uses a newer version of the V8 engine. 

The problem is that this new V8 engine (specifically anything higher than 6.2) is not well supported on FreeBSD. So I have tagged this as a "third party issue". See these upstream issues:
https://github.com/cowboyd/libv8/issues/253
https://bugs.chromium.org/p/chromium/issues/detail?id=857057

There are two approaches to fixing this:

1. Fall back to therubyracer (0.11.4) and libv8 (3.11.8.17).

2. Keep using the latest gem and V8 engine. Create an admin support document regarding this issue, saying: "while we don't officially support FreeBSD, if you change these two lines Zammad should work on FreeBSD. We'd love to support FreeBSD if it's within our abilities, but unfortunately this is an upstream issue outside of our control."

**Comment by MacLemon** [comment]
Thanks for the suggestion on how to get it running. I'll try that as soon as I can.

**Comment by MacLemon** [comment]
Finally reserved the downtime to check this out.
TL;DR: We tried and partially failed.

The correstponding part of the `Gemfile` looks like this:
```ruby
# asset handling - javascript execution for e.g. linux
gem 'execjs'
#gem 'libv8'
#gem 'mini_racer'
gem 'libv8', '3.11.8.0'
gem 'therubyracer'
```

We managed to get past the installation of `Using libv8 3.11.8.0` which does work, but then intalling `therubyracer` doesn't seem to work.

```
Fetching therubyracer 0.11.0
Installing therubyracer 0.11.0 with native extensions
Gem::Ext::BuildError: ERROR: Failed to build gem native extension.

    current directory: /usr/local/lib/ruby/gems/2.5/gems/therubyracer-0.11.0/ext/v8
/usr/local/bin/ruby25 -I /usr/local/lib/ruby/site_ruby/2.5 -r ./siteconf20190715-33530-17ztxun.rb extconf.rb
checking for -lpthread... yes
*** extconf.rb failed ***
Could not create Makefile due to some reason, probably lack of necessary
libraries and/or headers.  Check the mkmf.log file for more details.  You may
need configuration options.

Provided configuration options:
        --with-opt-dir
        --without-opt-dir
        --with-opt-include
        --without-opt-include=${opt-dir}/include
        --with-opt-lib
        --without-opt-lib=${opt-dir}/lib
        --with-make-prog
        --without-make-prog
        --srcdir=.
        --curdir
        --ruby=/usr/local/bin/$(RUBY_BASE_NAME)25
        --with-pthreadlib
        --without-pthreadlib
        --enable-debug
        --disable-debug
/usr/local/lib/ruby/gems/2.5/gems/libv8-3.11.8.0/lib/libv8.rb:30:in `block in libv8_objects': no libv8 object base (RuntimeError)
        from /usr/local/lib/ruby/gems/2.5/gems/libv8-3.11.8.0/lib/libv8.rb:29:in `map'
        from /usr/local/lib/ruby/gems/2.5/gems/libv8-3.11.8.0/lib/libv8.rb:29:in `libv8_objects'
        from /usr/local/lib/ruby/gems/2.5/gems/therubyracer-0.11.0/ext/v8/build.rb:51:in `build_with_rubygem_libv8'
        from extconf.rb:20:in `<main>'

To see why this extension failed to compile, please check the mkmf.log which can be found here:

  /usr/local/lib/ruby/gems/2.5/extensions/amd64-freebsd-11/2.5/therubyracer-0.11.0/mkmf.log

extconf failed, exit code 1

Gem files will remain installed in /usr/local/lib/ruby/gems/2.5/gems/therubyracer-0.11.0 for inspection.
Results logged to /usr/local/lib/ruby/gems/2.5/extensions/amd64-freebsd-11/2.5/therubyracer-0.11.0/gem_make.out

An error occurred while installing therubyracer (0.11.0), and Bundler cannot continue.
Make sure that `gem install therubyracer -v '0.11.0' --source 'https://rubygems.org/'` succeeds before bundling.

In Gemfile:
  therubyracer
```

**Comment by cesardi** [comment]
Stumbled into this same problem today. Solved changing versions in `Gemfile` to this:

```
gem 'libv8', '~> 3.16.14.15'
gem 'therubyracer', '~> 0.12.3'
```

Using `rvm` from rvm.io (not from ports) on FreeBSD 12-Release and Zammad-latest. 
Also using `gmake` like `export MAKE="gmake"` instead of `make`.

Partial output:

```
...
Using libv8 3.16.14.19 (was 6.7.288.46.1)
Using memoizable 0.4.2
Using mime-types-data 3.2019.0331
Using mime-types 3.2.2
Using multi_xml 0.6.0
Using mysql2 0.4.10
Using naught 1.1.0
Using net-ldap 0.16.1
Using netrc 0.11.0
Using oauth 0.5.4
Using oauth2 1.4.1
Using omniauth 1.9.0
Using omniauth-oauth2 1.6.0
Using omniauth-facebook 5.0.0
Using omniauth-github 1.3.0
Using omniauth-gitlab 2.0.0
Using omniauth-google-oauth2 0.6.1
Using omniauth-linkedin-oauth2 1.0.0
Using omniauth-microsoft-office365 0.0.7
Using omniauth-oauth 1.1.0
Using omniauth-rails_csrf_protection 0.1.2
Using omniauth-twitter 1.4.0
Using omniauth-weibo-oauth2 0.5.2
Using puma 3.12.1
Using sprockets 3.7.2
Using sprockets-rails 3.2.1
Using rails 5.2.3
Using rails-observers 0.1.5
Using raindrops 0.19.0
Using rchardet 1.8.0
Using ref 2.0.0
Using rest-client 2.0.2
Using rszr 0.5.2
Using rubyntlm 0.1.2 from https://github.com/wimm/rubyntlm (at master@5396963)
Using sassc 2.0.1
Using tilt 2.0.9
Using sassc-rails 2.1.1
Using simple_oauth 0.3.1
Using slack-notifier 2.3.2
Using telegramAPI 1.4.2
Using telephone_number 1.4.0
Fetching therubyracer 0.12.3
Installing therubyracer 0.12.3 with native extensions
...
```
Hope it helps.

**Comment by cesardi** [comment]
Update: 
`bundle install` passed using 
```
gem 'libv8', '~> 3.16.14.15'
gem 'therubyracer', '~> 0.12.3'
```
but `rake assets:precompile` fails. 
`autoprefixer-rails-9.6.1` does not support `therubyracer`

Output:
```
[zam@zammad ~]$ rake assets:precompile
Yarn executable was not detected in the system.
Download Yarn at https://yarnpkg.com/en/docs/install
rake aborted!
ExecJS::RubyRacerRuntime is not supported. Please replace therubyracer with mini_racer in your Gemfile or use Node.js as ExecJS runtime.
/usr/local/rvm/gems/ruby-2.5.5/gems/autoprefixer-rails-9.6.1/lib/autoprefixer-rails/processor.rb:143:in `runtime'
/usr/local/rvm/gems/ruby-2.5.5/gems/autoprefixer-rails-9.6.1/lib/autoprefixer-rails/processor.rb:36:in `process'
/usr/local/rvm/gems/ruby-2.5.5/gems/autoprefixer-rails-9.6.1/lib/autoprefixer-rails/sprockets.rb:20:in `run'
/usr/local/rvm/gems/ruby-2.5.5/gems/autoprefixer-rails-9.6.1/lib/autoprefixer-rails/sprockets.rb:14:in `call'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-3.7.2/lib/sprockets/processor_utils.rb:75:in `call_processor'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-3.7.2/lib/sprockets/processor_utils.rb:57:in `block in call_processors'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-3.7.2/lib/sprockets/processor_utils.rb:56:in `reverse_each'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-3.7.2/lib/sprockets/processor_utils.rb:56:in `call_processors'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-3.7.2/lib/sprockets/loader.rb:134:in `load_from_unloaded'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-3.7.2/lib/sprockets/loader.rb:60:in `block in load'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-3.7.2/lib/sprockets/loader.rb:317:in `fetch_asset_from_dependency_cache'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-3.7.2/lib/sprockets/loader.rb:44:in `load'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-3.7.2/lib/sprockets/cached_environment.rb:20:in `block in initialize'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-3.7.2/lib/sprockets/cached_environment.rb:47:in `load'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-3.7.2/lib/sprockets/bundle.rb:23:in `block in call'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-3.7.2/lib/sprockets/utils.rb:200:in `dfs'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-3.7.2/lib/sprockets/bundle.rb:24:in `call'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-3.7.2/lib/sprockets/processor_utils.rb:75:in `call_processor'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-3.7.2/lib/sprockets/processor_utils.rb:57:in `block in call_processors'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-3.7.2/lib/sprockets/processor_utils.rb:56:in `reverse_each'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-3.7.2/lib/sprockets/processor_utils.rb:56:in `call_processors'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-3.7.2/lib/sprockets/loader.rb:134:in `load_from_unloaded'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-3.7.2/lib/sprockets/loader.rb:60:in `block in load'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-3.7.2/lib/sprockets/loader.rb:317:in `fetch_asset_from_dependency_cache'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-3.7.2/lib/sprockets/loader.rb:44:in `load'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-3.7.2/lib/sprockets/cached_environment.rb:20:in `block in initialize'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-3.7.2/lib/sprockets/cached_environment.rb:47:in `load'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-3.7.2/lib/sprockets/base.rb:66:in `find_asset'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-3.7.2/lib/sprockets/base.rb:73:in `find_all_linked_assets'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-3.7.2/lib/sprockets/manifest.rb:134:in `block in find'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-3.7.2/lib/sprockets/manifest.rb:133:in `each'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-3.7.2/lib/sprockets/manifest.rb:133:in `find'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-3.7.2/lib/sprockets/manifest.rb:186:in `compile'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-rails-3.2.1/lib/sprockets/rails/task.rb:68:in `block (3 levels) in define'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-3.7.2/lib/rake/sprocketstask.rb:147:in `with_logger'
/usr/local/rvm/gems/ruby-2.5.5/gems/sprockets-rails-3.2.1/lib/sprockets/rails/task.rb:67:in `block (2 levels) in define'
/usr/local/rvm/gems/ruby-2.5.5/gems/rake-12.3.2/exe/rake:27:in `<top (required)>'
/usr/local/rvm/gems/ruby-2.5.5/bin/ruby_executable_hooks:24:in `eval'
/usr/local/rvm/gems/ruby-2.5.5/bin/ruby_executable_hooks:24:in `<main>'
Tasks: TOP => assets:precompile
(See full trace by running task with --trace)
```

**Comment by ntinti** [comment]
Sad that Zammad does not work an FreeBSD anymore. Is there no alternative to "libv8" and "mini-racer"? Do we go to OTRS again?

**Comment by thorsteneckel** [comment]
Hey there Zammad FreeBSD users 👋 Your responses to this issue somehow slipped through my hands. Anyhow, I had a look [at the commit where we switched from `therubyracer` to `mini_racer`](https://github.com/zammad/zammad/commit/3acfa4a277c29a0eb3b7bff83fde1a60718a9746). We did this because `mini_racer` is way more up to date in its `v8` runtime dependency and a bit faster. But both should be feature compatible with each other.
That's why I'd like to test something out. Unfortunately I have no opportunity to run FreeBSD, because the Docker image requires a FreeBSD host which we don't have. That's where I need you folks :) Anyone interested in helping get this thing running on your beloved FreeBSD machines? Drop us a mail to support@zammad.com and refer to this issue and me. Looking forward to hear from you. 🚀

**Comment by DrDBanner** [comment]
Is this still true to the latest release?

**Comment by MrGeneration** [comment]
🤷‍♂️ Nobody at Zammad I know is FreeBSD user so testing this is currently out of scope. 
As this issue hasn't received an update at this moment, it proberbly hasn't moved in any direction.

Please see the last comment from Thorsten, maybe you can help with that.

**Comment by Subito** [comment]
@DrDBanner yes, as far as I know everything >2.9 is currently not supported. We stopped trying after a few versions, though. We were in contact with Thorsten, but unfortunatly there is no fix possible atm, if I understood correctly.

**Comment by thorsteneckel** [comment]
Hey there! Sorry folks it slipped my mind to update this issue. I had contact with multiple FreeBSD users and we tried to make it work but weren't able to. Unfortunately. The issue is that we'd need to support both gems `ruby_racer` and `mini_racer` which doesn't work. However, it would be only a workaround as @SummerStorm already [pointed out](https://github.com/zammad/zammad/issues/2616#issuecomment-504888869) in much more detail.
We came to the conclusion that a third party would offer a dedicated Zammad package for FreeBSD and we would support where we can since we have no FreeBSD knowledge and/or infrastructure in house. However, it never came so far. The offer still stands! If anyone is willing to provide such a package we're happy to help where we can. Please reach out to support at zammad.com and refer to this issue.

**Comment by ntinti** [comment]
Any News? Or how does "Horst" install Zammad 3.4 on FreeBSD 12.1 ?
https://community.zammad.org/t/locking-a-record-with-unpersisted-changes-is-not-supported/4861

**Comment by kapfenho** [comment]
Sorry - I've passed this workaround to MacLemon, but we forgot to post them here.

Attached you can find the steps: [upgrade-zammad-3.4.0.txt](https://github.com/zammad/zammad/files/5322391/upgrade-zammad-3.4.0.txt)

Summary: the problems are in the asset compilation step, where the final
javascript and css files are created.

You can execute this step on another OS and copy the results - the files
should always look the same:

	node_modules/
	public/assets/
	yarn.lock

**Comment by ntinti** [comment]
Hey - your little instruction helps me a lot. Thank you. Zammad 3.7 runs now in a ZFS FreeBSD 12.2 jail. I want my database and proxy server on other hosts than zammad and the Debian package isn't an alternative for such configuration. And manual installation on ubuntu 20 sucks... 
❤️ FreeBSD for it's flexibility and clearness.

**Comment by MrGeneration** [comment]
The source installation documentation (for supported systems) will be improved in the future. That's currently undergroing QA.
https://github.com/zammad/zammad-documentation/pull/156/files#diff-c4ce3ec78b956bc6801c0a1adc11253b24534958636adfc57c59259491c510bb

**Comment by thorsteneckel** [comment]
Heads up fellow FreeBSD users! Good news: We will remove the `mini_racer` dependency with the next major release 5.0 of Zammad 😍  Instead we will require node.js as a build dependency (just for `rake assets:precompile` for now) which is available for FreeBSD. We're far from having a concrete date yet.

However, what we have is a workaround that might help you a bit: [We found a way to make the `mini_racer` dependency optional with a fallback to a local node.js](https://github.com/zammad/zammad/commit/236420da75888251f968a57d23353997451b88db) 👨‍🔬 To make it work make sure node.js is installed on your servers. We don't know the minimal version yet but node.js 10.13 (from 2019) worked in our tests. After that run `bundle install` as usual but add the `--without mini_racer` flag to your call.

Please note that this will be available with the next upcoming minor release `4.1` in a few weeks. It's a sneak peek and won't be documented and only partially supported till 5.0.

For further (technical) questions regarding this topic please refer to the [community board](http://community.zammad.org/).

**Comment by ntinti** [comment]
😻 Cool, very cool!

**Comment by ntinti** [comment]
Just created Zammad on FreeBSD 12.2 from develop branch without problems 💯 👍
